src/app/table/data-table/terra-data-table.component.html

```diff
@@ -81,7 +81,9 @@
                         </div>
                         
                         <div *ngSwitchCase="'TerraRefTypeInterface'">
-                            <a href="{{cell.data.type}}:{{cell.data.value}}" (click)="$event.stopPropagation()">{{cell.data.value}}</a>
+                            <a *ngIf="cell.data.type === 'mailto' || cell.data.type === 'tel'" href="{{cell.data.type}}:{{cell.data.value}}" (click)="$event.stopPropagation()">{{ cell.data.caption || cell.data.value}}</a>
+                            <a *ngIf="cell.data.type === 'fct'" href="javascript:void(0);" (click)="cell.data.value(); $event.stopPropagation();">{{cell.data.caption}}</a>
+                            <a *ngIf="cell.data.type === 'url'" href="{{cell.data.value}}" target="{{cell.data.target || '_self'}}" (click)="$event.stopPropagation()">{{cell.data.caption || cell.data.value}}</a>
```

Comment:
bitte versuche bei allen auch die Enum zu nutzen. Z.B. `cell.data.type === refTypeEnum.url`. Dazu musst du in der Component eine private Variable wie folgt anlegen:`private refTypeEnum = TerraRefTypeEnum;`